js/library.js

```diff
@@ -33,6 +33,134 @@
     });
 })(wp.media.view);
 
+function createPopup() {
+
+    var overlay = document.createElement('div');
```

Comment:
Just a suggestion.

You can render this structure here right after here if the user is not PRO: https://github.com/Codeinwp/visualizer/blob/797064cd5c63ebf0897714cadbbbeae0f5171d3c/classes/Visualizer/Render/Library.php#L253

By default, it will have `display: none`, and with JS, you can toggle it.